Client/metrics.yaml

```diff
@@ -2802,3 +2802,16 @@ adjust:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: "2023-01-01"
+migration:
+  image_sd_cache_cleanup:
+    type: counter
+    description: |
+        Counts the number of times a user runs the
+        sd web image library cache cleanup
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-4321
```

Comment:
I think we are trying to remove the JIRA tickets and use the PR link in both `bugs` and `data_reviews` as per preview review I received

Reply:
Oh for sure, this is actually better